docs/explanation/parts.rst

```diff
@@ -0,0 +1,263 @@
+Parts
+=====
+
+In the Craft Parts framework, parts are descriptions of the components to be
+build for deployment, either individually or as part of a larger build
```

Comment:
built* (typo)

Reply:
Yes, that's a good point. "Build" gets used a lot for the build step but also for the whole project.

Reply:
Sometimes I use "payload" to refer to the final content of the snap file.